src/lib/modules/directory/components/ProviderProfile/ProviderProfile.tsx

```diff
@@ -195,368 +195,388 @@ export function ProviderProfile({
     ).sort();
 
     return (
-        <ProfileContainer
-            sx={{
-                minHeight:
-                    ELEMENT_DIMENSIONS.headerHeight +
-                    ELEMENT_DIMENSIONS.avatar * 2.5,
-                height: '100%',
-                overflowY: 'auto',
-                paddingBottom: isSmallScreen && member ? 30 : 0,
-            }}
-        >
-            <ProfileBanner>
-                <PageHeader
-                    type="abstract1"
-                    title={''}
-                    containerProps={{
-                        sx: {
-                            height: ELEMENT_DIMENSIONS.headerHeight,
-                        },
-                    }}
-                />
-                <Avatar
-                    size={ELEMENT_DIMENSIONS.avatar}
-                    src={profileImageUrl ?? undefined}
-                    sx={{ position: 'absolute', top: '50%' }}
-                />
-            </ProfileBanner>
-            <ProfileWrapper>
-                <ProfileContent
-                    sx={{
-                        marginTop:
-                            ELEMENT_DIMENSIONS.avatar *
-                            ELEMENT_DIMENSIONS.contentMarginTop,
-                    }}
-                >
-                    <ProviderTitle spacing={-2}>
-                        <ProviderNameContainer
-                            direction={'row'}
-                            alignItems="baseline"
-                            spacing={2}
+        <ScrollBox>
+            <ProfileContainer
+                sx={{
+                    minHeight:
+                        ELEMENT_DIMENSIONS.headerHeight +
+                        ELEMENT_DIMENSIONS.avatar * 2.5,
+                    paddingBottom: 30,
+                }}
+            >
+                <ProfileBanner>
+                    <PageHeader
+                        type="abstract1"
+                        title={''}
+                        containerProps={{
+                            sx: {
+                                height: ELEMENT_DIMENSIONS.headerHeight,
+                            },
+                        }}
+                    />
+                    <Avatar
+                        size={ELEMENT_DIMENSIONS.avatar}
+                        src={profileImageUrl ?? undefined}
+                        sx={{ position: 'absolute', top: '50%' }}
+                    />
+                </ProfileBanner>
+                <ProfileWrapper>
+                    <ProfileContent
+                        sx={{
+                            marginTop:
+                                ELEMENT_DIMENSIONS.avatar *
+                                ELEMENT_DIMENSIONS.contentMarginTop,
+                        }}
+                    >
```

Comment:
This looks like a huge change, but all it does is wrap the entire existing JSX in `<ScrollBox></Scrollbox>`. That causes reformatting that looks like it changes the whole JSX portion of the component, but, functionally, it's the same. Same goes for the `directory.tsx` page